### ISSUE TITLE
Keyboard focus doesn't work as expected

### ISSUE THREAD

**Issue by rahulporuri** [fossunited/forklore#43]
Steps to reproduce
------------------
- open https://forklore.in
- start pressing tab or shift+tab. Start at the top of the page i.e. "forklore_" > "Get Featured" > "FOSS United Grants" > "Light/Dark" theme toggle > "FOSS United" link
- pressing tab one more time should select the first maintainer (at the moment "ayan singh") but instead keyboard/tab focus moves to the project within the maintainer (at the moment "scancode-toolkit")
- continuing to tab shows that the "maintainer" page/link cannot be brought to focus using tabs - the focus moves from project to project until the page ends
